notebooks/MAST/exploring_jwst_transmission/comparing_spectral_types_jwst.ipynb

```diff
@@ -0,0 +1,7074 @@
+{
```

Comment:
<p>I would put the K star first, so that users can see the necessity of this function/filtering for unique files</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>